packages/bot-web-ui/src/pages/dashboard/announcements/announcements.tsx

```diff
@@ -0,0 +1,185 @@
+import { IconAnnounce, MessageAnnounce, TitleAnnounce } from './announcement-components';
+
+type TAnnouncements = {
+    is_mobile?: boolean;
+    handleTabChange: (item: number) => void;
+};
+
+const Announcements = ({ is_mobile, handleTabChange }: TAnnouncements) => {
+    const [isAnnounceDialogOpen, setIsAnnounceDialogOpen] = useState(false);
+    const [isOpenAnnounceList, setIsOpenAnnounceList] = React.useState(false);
+    const [amountAnnounce, setAmountAnnounce] = useState({} as Record<string, boolean>);
+    const buttonRef = React.useRef<HTMLButtonElement>(null);
+    const wrapperRef = React.useRef<HTMLDivElement>(null);
+    const accumulator_announcement = ANNOUNCEMENTS.ACCUMULATOR_ANNOUNCE;
+    const is_active_announce_1 = amountAnnounce?.announce_1;
+
+    const handleClickOutside = (event: React.MouseEvent<HTMLButtonElement>) => {
+        if (
+            !wrapperRef?.current?.contains(event.target as HTMLButtonElement) &&
+            !buttonRef?.current?.contains(event.target as HTMLButtonElement)
+        ) {
+            setIsOpenAnnounceList(false);
+        }
+    };
+
+    useEffect(() => {
+        document.addEventListener('click', handleClickOutside);
+        return () => {
+            document.removeEventListener('click', handleClickOutside);
+        };
+    }, []);
+
+    const handleAnnounceSubmit = (data: Record<string, boolean>) => {
+        setAmountAnnounce(data);
+        localStorage?.setItem('bot-announcements', JSON.stringify(data));
+    };
+
+    const announcements = [
+        {
+            icon: <IconAnnounce announce={is_active_announce_1} />,
+            title: (
+                <TitleAnnounce title={localize('Accumulators is now on Deriv Bot')} announce={is_active_announce_1} />
+            ),
+            message: (
+                <MessageAnnounce
+                    message={localize('Boost your trading strategy with Accumulators.')}
+                    date='20 July 2024 00:00 UTC'
+                    announce={is_active_announce_1}
+                />
+            ),
+            buttonAction: () => {
+                setIsAnnounceDialogOpen(true);
+                setIsOpenAnnounceList(prev => !prev);
+                handleAnnounceSubmit({ ...amountAnnounce, announce_1: false });
+            },
+            actionText: '',
+        },
+    ];
+
+    useEffect(() => {
+        let data: Record<string, boolean> | null = null;
+        data = JSON.parse(localStorage.getItem('bot-announcements') ?? '{}');
+
+        if (data && Object.keys(data).length !== 0) {
+            setAmountAnnounce(data);
+        } else {
+            const obj_announcements = Object.fromEntries(
+                Array.from({ length: announcements.length }, (_, i) => [`announce_${i + 1}`, true])
+            );
+            setAmountAnnounce(obj_announcements);
+            localStorage?.setItem('bot-announcements', JSON.stringify(obj_announcements));
+        }
+    }, [announcements.length]);
```

Comment:
@maryia-matskevich-deriv the `announcements` is just a const and not a state. It doesn't have any effect on the dependency I believe.

Reply:
@shafin-deriv Yes, we can remove it, I added it since it was a suggestion of warning, but I agree that without it, it is cleaner in my opinion